### DISCUSSION TITLE
Creating LARGE animated scenery objects

### DISCUSSION THREAD

**Discussion by IceKnight366** [OpenRCT2/OpenRCT2#15779] [Feature requests]
Windows 7
Version:. 0.2.6

I'm big into creating custom content, particularly animated objects.  But anyone who has created custom animated objects knows that you are limited in a number of ways, one of those being that you can only created small objects if you want those objects to be animated.  On top of all the amazing limits that will be lifted once the new save format is eventually released, is it possible to create custom animated **large** scenery objects?  Perhaps update the existing v1.23d object editor to enable animated large scenery?  This could open up the customization of the game in a large way and get more people interested in making custom scenery.

**Comment by Gymnasiast** [comment]
We have no way of updating the Object Editor - we don't have the source code, and it has also been known to generate slightly faulty objects.

However, we now use JSON objects, which are user-readable and not binary.

**Comment by IceKnight366** [comment]
Has JSON been used to add any custom objects into ORCT2 yet?  If so I'd like to see them and perhaps a guide on how it was done.  I looked up a youtube video and it seems it's not visual like the object editor is, but more code based - like a completely new beast.
Nevertheless, I guess 1.23d objector editor will become obsolete at some point, so if it's possible to animated large scenery objects it will have to be done with JSON?

**Comment by Gymnasiast** [comment]
With JSON or with a new tool. The advantage is that the format is fully known and that we are around to explain stuff - that was never the case for the Object Editor, which had to implement support based on what scarce information the community had gathered themselves back in the day.

There aren't any custom JSON objects _yet_ - the format is still evolving a bit, there isn't an easy mechanism to spread them yet, and SV6 does not support them.
